.circleci/config.yml

```diff
@@ -70,13 +63,18 @@ jobs:
       - run: git submodule sync && git submodule update --init
       - restore_cache:
           keys:
-            - ccache
+            - ccache-release-linux-x86_64-{{ .Branch }}
```

Comment:
Does circleci grab the first cache that exists?

Reply:
> If there are multiple matches, the most recently generated cache will be used.

yep.

Reply:
Yeah - it's a series of fallbacks - this should cause it to grab the most recent cache that matches this particular pattern.